### PR TITLE
[sc-460] Update tests to show HVM.log output

### PR DESCRIPTION
**Pull request by imaqtkatt**

### CODE REVIEW

File: tests/golden_tests.rs [hunk 1]
```diff
@@ -141,19 +145,21 @@ fn linear_readback() {
...
   run_golden_test_dir_multiple(function_name!(), &[
-    (&|code, path| {
-      let book = do_parse_book(code, path)?;
-      // 1 million nodes for the test runtime. Smaller doesn't seem to make it any faster
-      let (res, info) =
-        run_book(book, 1 << 24, RunOpts::lazy(), WarningOpts::deny_all(), CompileOpts::heavy(), None)?;
-      Ok(format!("{}{}", display_readback_errors(&info.readback_errors), res))
+    (&|_code, path| {
+      let output = std::process::Command::new(env!("CARGO_BIN_EXE_hvml"))
+        .args(["run", path.to_str().unwrap(), "-Dall", "-Oall", "-L"])
```
Comment (developedby) [inline tests/golden_tests.rs:150]: i think we have to add -Dall to the dictionary here

**Review by developedby — CHANGES_REQUESTED** [review]
Looks like one of the tests has changed

**Comment by LunaAmora** [comment]
> Looks like one of the tests has changed

This is the same issue as we discoursed on discord [here](https://discord.com/channels/912426566838013994/1006963940649087018/1212760931231072306).

We need a different task implementing this [solution](https://discord.com/channels/912426566838013994/1006963940649087018/1212763229516726292) you suggested to fix it.

**Review by LunaAmora — APPROVED** [review]

**Comment by developedby** [comment]
> > Looks like one of the tests has changed
> 
> This is the same issue as we talked about on discord [here](https://discord.com/channels/912426566838013994/1006963940649087018/1212760931231072306).
> 
> We need a different task implementing this [solution](https://discord.com/channels/912426566838013994/1006963940649087018/1212763229516726292) you suggested to fix it.

I thought we solved that already, but I guess not. @imaqtkatt can you open a new task for that?

**Review by developedby — APPROVED** [review]

**Review by LunaAmora — APPROVED** [review]

**Pull request merged**